File: Functions/notes.c
// Julia Properzi, Function notes in C
#include <stdio.h>
int num;
char name[50], place[50], verb[50];
int add(int numOne, int numTwo){
    return "%d\n", numOne+ numTwo;
}
const char* word(char type[50]){
    char choice[50];
    printf("Please give me a %s:\n",type);
    scanf("%s", choice);
    return choice;
}

int main(void){
    //printf("Please tell me a number:\n");
   // scanf("%d", num);
   // add(num,10);
    //add(8,1);
    printf("%d",add(72,5));
    return 0;
}

File: Functions/notes.py
# Julia Properzi, Functions Notes python

#funcions hold actions to be reused

#def add():
   # numOne = int(input("Please tell me a number:\n"))
   # numTwo = int(input("Tell me another number: \n"))

#number = int(input("Please tell me a number:\n"))
#def add(numOne, numTwo): #parameter set the name of the variable(just for the funcion)
 #   return numOne + numTwo
    #print (numOne+numTwo)

#print (add(number,21)) #arguments set the value of the variable just for that instance of the function
#print(add(8,12))
#addition = add(6,4)
#print(add(11,10000))
#add(3,67)

def values(type):
    return input(f"Please give me a {type}:\n")
name =values("name")
place =values("place")
verb = values("verb")

print(f"{name} was really fast getting to {place} because they were {verb} the whole way there.")